app/(auth)/signin.js

```diff
@@ -46,7 +46,7 @@ export default function SignIn() {
         await signInWithEmailAndPassword(auth, email, password);
       } catch (e) {
         console.log(e);
-        showDialog("Error", getErrorString(e));
+        showDialog("Error", "Authentication Error"); //maybe could be more specific about only invalid email and invalid password errors
```

Comment:
```suggestion
        showDialog("Error", "Authentication Error. Invalid Email or Password was entered."); 
```
^^ I am unsure of what other errors could be thrown. Additionally, this could just be something we add into the firebaseErrors object in constants.